WordPress-Extra/ruleset.xml

```diff
@@ -2,6 +2,22 @@
 <ruleset name="WordPress Extra">
 	<description>Best practices beyond core WordPress Coding Standards</description>
 
+	<rule ref="Generic.PHP.DeprecatedFunctions"/>
+	<rule ref="Generic.PHP.ForbiddenFunctions"/>
+	<rule ref="Generic.Functions.CallTimePassByReference"/>
+	<rule ref="Generic.Formatting.DisallowMultipleStatements"/>
+	<rule ref="Generic.Formatting.MultipleStatementAlignment"/>
```

Comment:
I really don't think we should include `MultipleStatementAlignment`. It's really annoying, for instance:

![class-https-resource-proxy_php_-_wordpress-develop_-____projects_vvv_www_wordpress-develop_](https://cloud.githubusercontent.com/assets/134745/7544125/4a417fda-f57f-11e4-8db0-3f4536f7be70.png)